src/library_pthread.js

```diff
@@ -211,8 +211,8 @@ var LibraryPThread = {
 #endif
       // Call thread init functions (these are the emscripten_tls_init for each
       // module loaded.
-      for (var i in PThread.tlsInitFunctions) {
-        PThread.tlsInitFunctions[i]();
+      for (const f of PThread.tlsInitFunctions) {
```

Comment:
How about `for (var i in PThread.tlsInitFunctions.keys())` ?

(Also we haven't started moving the codebase to `let`/`const` yet.

Reply:
Sure, can adapt 👍 Seems to do exactly what we want.